src/plugins/preauth/pkinit/pkinit_srv.c

```diff
@@ -449,7 +451,7 @@ pkinit_server_verify_padata(krb5_context context,
 
     switch ((int)data->pa_type) {
     case KRB5_PADATA_PK_AS_REQ:
-        pkiDebug("processing KRB5_PADATA_PK_AS_REQ\n");
+        TRACE_PKINIT_SERVER_PADATA_VERIFY(context, "KRB5_PADATA_PK_AS_REQ");
```

Comment:
We shouldn't call the same TRACE macro twice; they are supposed to be usable as DTrace tracing points if the macros are changed.  So use two different macros here, not the same one with a string literal parameter.